packages/flutter/test/rendering/wrap_test.dart

```diff
@@ -12,17 +12,146 @@ void main() {
     expect(
       renderWrap.toStringDeep(minLevel: DiagnosticLevel.info),
       equalsIgnoringHashCodes(
-        'RenderWrap#00000 NEEDS-LAYOUT NEEDS-PAINT DETACHED\n'
-        '   parentData: MISSING\n'
-        '   constraints: MISSING\n'
-        '   size: MISSING\n'
-        '   direction: horizontal\n'
-        '   alignment: start\n'
-        '   spacing: 0.0\n'
-        '   runAlignment: start\n'
-        '   runSpacing: 0.0\n'
-        '   crossAxisAlignment: 0.0\n'
-      ),
+          'RenderWrap#00000 NEEDS-LAYOUT NEEDS-PAINT DETACHED\n'
+          '   parentData: MISSING\n'
+          '   constraints: MISSING\n'
+          '   size: MISSING\n'
+          '   direction: horizontal\n'
+          '   alignment: start\n'
+          '   spacing: 0.0\n'
+          '   runAlignment: start\n'
+          '   runSpacing: 0.0\n'
+          '   crossAxisAlignment: 0.0\n'),
     );
   });
+
+  test(
+    'Compute intrinsic height test',
+    () {
+      final List<RenderBox> children = <RenderBox>[
+        RenderConstrainedBox(
+          additionalConstraints:
+              const BoxConstraints(minWidth: 80, minHeight: 80),
```

Comment:
use trailing commas to format this:

```suggestion
          additionalConstraints: const BoxConstraints(
            minWidth: 80,
            minHeight: 80,
          ),
```

Reply:
Same for the next two.